application-xpoll-ui/src/main/resources/XPoll/XPollSheet.xml

```diff
@@ -398,7 +404,13 @@
 .info
 {
   float: left;
-}</code>
+}
+
+#xpollSaveForm .noBorder {
+  border-right: solid 1px #FFF;
```

Comment:
The classical order for border shorthand is (width style color), see https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties#Border_Properties